assets/src/js/admin/onboarding-wizard/app/steps/introduction/index.js

```diff
@@ -4,12 +4,19 @@ const { __ } = wp.i18n;
 // Import styles
 import './style.scss';
 
+// Import utilities
+import { genereateFormPreviewWithOnboardingAPI } from '../../../utils';
```

Comment:
Oop! Let's fix this typo in the function name

Reply:
dca2582c9